src/app/pages/MediaArticlePage/MediaArticlePage.tsx

```diff
@@ -176,6 +188,7 @@ const MediaArticlePage = ({ pageData }: MediaArticlePageProps) => {
     metadata: { atiAnalytics },
   } = pageData;
 
+  if (atiAnalytics) atiAnalytics.contentType = 'article-media-asset';
```

Comment:
I think we should perhaps do this in the BFF if possible. The ATI analytics metadata is constructed in our Article page route in there, so it would be best to keep all that logic in the same place.